homeassistant/components/wemo/__init__.py

```diff
@@ -149,7 +149,12 @@ async def async_add_unique_device(
         if wemo.serialnumber in self._added_serial_numbers:
             return
 
-        coordinator = await async_register_device(hass, self._config_entry, wemo)
+        try:
+            coordinator = await async_register_device(hass, self._config_entry, wemo)
+        except pywemo.PyWeMoException as err:
+            _LOGGER.error("Unable to add WeMo %s %s: %s", repr(wemo), wemo.host, err)
```

Comment:
Given what you wrote I suppose this would print in log every time it runs. Can it be avoided?

Reply:
Yes. Fixed in [42d0255](https://github.com/home-assistant/core/pull/91621/commits/42d025553a66dacd5b4851a937cac4b8450f38be)